pages/instructorAssessmentManualGrading/instructorAssessmentManualGrading.ejs

```diff
@@ -23,13 +33,13 @@
                 <th></th>
                 <th>QID</th>
                 <th>Points</th>
-                <th>Number of Submissions</th>
+                <th>Submissions to grade</th>
                 <th></th>
               </tr>
             </thead>
             <tbody>
               <% questions.forEach(function(question, i) { %>
-              <tr>
+              <tr <% if (question.num_instance_questions_assigned > 0) { %>class="bg-light"<% } %> >
```

Comment:
This is a very subtle highlighting of questions assigned to the current user. I'm happy to discuss something stronger or removing it altogether.